src/libs/libchdr/lzma/CpuArch.h

```diff
@@ -83,6 +83,14 @@ MY_CPU_LE_UNALIGN means that CPU is LITTLE ENDIAN and CPU supports unaligned mem
 #endif
 
 
+#if  defined(__loongarch64) \
+  || defined(__loongarch64__)
```

Comment:
Please just test for `__loongarch_lp64`. The `__loongarch64` name is deprecated while `__loongarch64__` has never been included in any platform spec.